recipes/leftbar/contact_row/index.js

```diff
@@ -0,0 +1,2 @@
+export { default as DtRecipeContactRow } from './contact_row.vue';
+export {} from './contact_row_constants';
```

Comment:
Same on the previous comment, where we are using DEFAULT_CONSTANTS?

Reply:
removed.